configs/body_factory/default/Makefile.am

```diff
@@ -44,5 +44,6 @@ dist_bodyfactory_DATA = \
 	timeout\#activity \
 	timeout\#inactivity \
 	transcoding\#unsupported \
-	urlrouting\#no_mapping
+	urlrouting\#no_mapping \
+        request\#uri_len_too_long
```

Comment:
indentation is not consistent

Reply:
this is odd, because I don't see the indentation issue in my workspace (and clang-format passes, so ;) )